source/mainnet/smart-contracts/guides/no-std.rst

```diff
@@ -26,12 +32,28 @@ This feature is enabled by default.
 
 To disable it, one must simply disable default features for the
 ``concordium-std`` in the dependencies of your module.
+You can update your `Cargo.toml` file by using:
 
 .. code-block:: rust
 
    [dependencies]
    concordium-std = { version: "=0.2", default-features = false }
 
+.. note::
+
+   To compile your smart contracts, a memory `allocator <https://docs.rs/concordium-std/6.0.0/concordium_std/#use-a-custom-allocator>`_ is used.
+   ``concordium-std`` version ``<6.0.0`` hard-coded the use of the `wee_alloc <https://docs.rs/wee_alloc/>`_ allocator.
+   In ``concordium-std`` version ``>=6.0.0``, ``wee_alloc`` is a feature and needs to be explicitly enabled.
+   When ``std`` feature is enabled, the allocator provided by the Rust standard library is used
+   by default but when the ``wee_alloc`` feature is enabled in addition, ``wee_alloc`` is used instead.
+   When ``no_std`` is used either ``wee_alloc`` must be enabled, or another global allocator
+   must be set in the smart contract. You can add the ``wee_alloc`` feature by using:
+
+   .. code-block:: rust
+
+      [features]
+      wee_alloc = ["concordium-std/wee_alloc"]
```

Comment:
I'm not sure what you mean to convey here, because there are two separate cases.

- They can add a feature to their contract to enable/disable wee_alloc
- They just enable wee_alloc.

I think we should describe both scenarios.